src/logic/__test__/calculator-operator.test.js

```diff
@@ -0,0 +1,56 @@
+// import { render, screen } from '@testing-library/react';
+import calculator from '../calculate';
+import operate from '../operate';
+
+/* CALCULATE.JS  */
+describe('Calculator.js Testing ', () => {
+  test('Test for button name as AC', () => {
+    const obj = {
+      total: 5,
+      next: 2,
+      operation: '',
+    };
+    const buttonName = 'AC';
+
+    const response = calculator(obj, buttonName);
+
+    expect(response.total).toBe(null);
+  });
+
+  test('Test for button name as = ', () => {
+    const obj = {
+      total: 5,
+      next: 2,
+      operation: '+',
+    };
+    const buttonName = '=';
+
+    const response = calculator(obj, buttonName);
+
+    expect(response.total).toBe('7');
+    expect(response.next).toBe(null);
+  });
+});
+
+/* OPERATE.JS  */
+describe('Test for Operate.js', () => {
+  test('operation for Addition', () => {
+    const numOne = 6;
+    const numTwo = 3;
+    const operation = '+';
+
+    const response = operate(numOne, numTwo, operation);
+
+    expect(response).toBe('9');
+  });
+
+  test('operation for Subtraction', () => {
+    const numOne = 6;
+    const numTwo = 3;
+    const operation = '-';
+
+    const response = operate(numOne, numTwo, operation);
+
+    expect(response).toBe('3');
+  });
+});
```

Comment:
- [ ] Same issue as above, please include tests for other operations too.

- [ ] One last thing, you've not used React Testing testing library anywhere to test/simulate user interactions. This is part of the project requirements and as quoted below:

  > - Use React Testing Library to simulate user interaction.

  ## Suggestion
  One user interaction we can implement is clicking a combination of keys and lastly the (=) key using the `fireEvent` method from `@testing-library/react` and then asserting that the result of that calculation is present on the page using the `getByText` method.
  
  This simple [GUIDE](https://www.pluralsight.com/guides/simulate-browser-events-react-react-testing-library) from Pluralsight can really help with this.

Reply:
Thanks for the guide